ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -172,6 +174,40 @@ public void setFontSize(ReactEditText view, float fontSize) {
         (int) Math.ceil(PixelUtil.toPixelFromSP(fontSize)));
   }
 
+  @ReactProp(name = ViewProps.FONT_FAMILY)
+  public void setFontFamily(ReactEditText view, String fontFamily) {
+    view.setTypeface(Typeface.create(fontFamily,view.getTypeface().getStyle()));
+  }
+
+  @ReactProp(name = ViewProps.FONT_WEIGHT)
+  public void setFontWeight(ReactEditText view, @Nullable String fontWeightString) {
```

Comment:
Given that this code is presumably copied from ReactTextShadowNode, would be nice to either factor into a common place they can both use or at least make a comment indicating that (probably in both places so that if someone updates one they know to update the other)